state-chain/traits/src/mocks/account_role_registry.rs

```diff
@@ -11,6 +11,8 @@ impl MockPallet for MockAccountRoleRegistry {
 }
 
 const ACCOUNT_ROLES: &[u8] = b"AccountRoles";
+pub const ALREADY_REGISTERED_ERROR: DispatchError =
+	DispatchError::Other("Account already registered");
```

Comment:
So in this case we don't want to take it out of the Other("string") and into a comment?

Reply:
As discussed, we decided that we can leave the cases where the string are used in tests only as is.